go/teams/transactions.go

```diff
@@ -420,6 +437,57 @@ func (tx *AddMemberTx) ReAddMemberToImplicitTeam(ctx context.Context, uv keybase
 	return nil
 }
 
+// AddMemberBySBS is very similar in what it does to addMemberByUPAKV2
+// (or AddMemberBy* family of functions), but it has easier job
+// because it only adds cryptomembers and fails on PUKless users. It
+// also completed inviteID for given invitee.
```

Comment:
`It also completed inviteID for given invitee.`
What did you mean?

Reply:
It signs in `completedInvites[invitee.inviteID] = uv` to the team chain. Let me rephrase that to something better.